template.js

```diff
@@ -1,18 +1,69 @@
+/* eslint-disable no-undef, prefer-template, no-useless-escape */
 const readme = require('@readme/cloudflare-worker');
 
 addEventListener('fetch', event => {
   event.passThroughOnException();
 
-  event.respondWith(respond(event));
+  if (determineRouting(event.request.url)) {
+    event.respondWith(respond(event));
+  }  else {
+    event.respondWith(fetch(event.request));
+  }
 });
 
 async function respond(event) {
   const { response, har } = await readme.fetchAndCollect(event.request);
 
-  event.waitUntil(readme.metrics('API_KEY', {
+  event.waitUntil(readme.metrics(INSTALL_OPTIONS.API_KEY, {
     id: response.headers.get('x-readme-id'),
     label: response.headers.get('x-readme-label'),
   }, event.request, har));
 
   return response;
 }
+
+function determineRouting(url) {
+  const {
+    protocol: reqProtocol,
+    host: reqDomain,
+    pathname: reqPath } = new URL(url);
```

Comment:
Will there be a browser compatibility issue with using `new URL`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

Reply:
<img width="1166" alt="Screen Shot 2019-08-02 at 11 38 02 AM" src="https://user-images.githubusercontent.com/8877659/62391559-03db6d80-b51a-11e9-8de5-4f8e47e56dd5.png">
Shouldn't be an issue.

Reply:
Do we officially support IE11 at all? Wonder if we should use a polyfill, or just note in the service worker description that it doesn't support non-Edge IE.

Reply:
No IE?

Reply:
Think the latter. IIRC, IE has been laughed at in the past.

Reply:
I think we should support it, unfortunately. We are a vendor and we should make this service available in the latest version of IE in the least.

Reply:
@dok Here's an interesting comment from a cloudflare dev:
```
Cloudflare Workers would live on our edge, and would be (subject to change) scoped to a particular route/path. Code isn't exposed on the client side, and thus you don't need client-side changes or even browser-side support for the Service Worker API (which is minimal, esp. on mobile!).
```

Reply:
So i don't think it's an issue, but worth testing. Do we even have windows machines or a machine with a VM setup that has IE? It doesn't seem like it's on the forefront of our QA

Reply:
Oh, interesting. They say they use V8, but not explicitly Node itself, so we should be good to use `URL` then.

https://developers.cloudflare.com/workers/about/how-workers-work/

Reply:
https://developers.cloudflare.com/workers/reference/

> Cloudflare Workers uses the V8 JavaScript engine from Google Chrome. The Workers runtime is updated at least once a week, to at least the version that is currently used by Chrome’s stable release. This means you can safely use latest JavaScript features, with no need for “transpilers”.

They also call out the URL API on this page, so we should be good!

Reply:
Forgot about that, thanks Jon!

Reply:
That is cool! Thanks for digging into it @essonjon @gratcliff

Reply:
We use https://www.browserstack.com/ for testing on IE

Reply:
Ahh cool, thanks Sean!